Sources/EduIDExpansion/Styling/UIColor+Styling.swift

```diff
@@ -15,7 +15,7 @@ extension UIColor {
     static let disabledGrayBackground = UIColor(named: "GrayDisabledBackground", in: Bundle.module, compatibleWith: nil)!
     static let grayGhost = UIColor(named: "GrayGhost", in: Bundle.module, compatibleWith: nil)!
     static let charcoalColor = UIColor(named: "CharcoalColor", in: Bundle.module, compatibleWith: nil)!
-    static let tertiary = UIColor(named: "TertiaryColor", in: Bundle.module, compatibleWith: nil)!
+    static let tertiaryColor = UIColor(named: "TertiaryColor", in: Bundle.module, compatibleWith: nil)!
```

Comment:
Minor: just curious, why add "Color". That is a color is obvious already, right?

Reply:
A matter of taste I think